flink-connectors/flink-orc/pom.xml

```diff
@@ -54,6 +54,14 @@ under the License.
 			<optional>true</optional>
 		</dependency>
 
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-connector-filesystem_${scala.binary.version}</artifactId>
+			<version>${project.version}</version>
+			<!-- Projects depending on this project, won't depend on flink-filesystem. -->
+			<optional>true</optional>
+		</dependency>
+
 		<dependency>
 			<groupId>org.apache.orc</groupId>
 			<artifactId>orc-core</artifactId>
```

Comment:
Current orc version is 1.5.X. Should we upgrade it as well?

Reply:
Yes. We can upgrade it. Will update.